src/components/PageNotFound/PageNotFound.jsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import './PageNotFound.scss';
+
+export const PageNotFound = () => (<h1 className="title"> PAGE NOT FOUND </h1>);
```

Comment:
```suggestion
export const PageNotFound = () => <h1 className="title"> PAGE NOT FOUND </h1>;
```
так має також спрацювати, но карще це записувати не в одну лінію коду